### PR TITLE
Avoid calling obsolete methods

### PR DESCRIPTION
**Pull request by sungam3r**
Rel: https://github.com/open-telemetry/opentelemetry.io/pull/2330

### CODE REVIEW

File: src/cartservice/src/Program.cs
Comment (cijothomas) [inline src/cartservice/src/Program.cs:45]: please update all packages to the newest RC.

Comment (sungam3r) [reply src/cartservice/src/Program.cs:45]: I would but cartservice.sln does not compile in `main`.

Comment (sungam3r) [reply src/cartservice/src/Program.cs:45]: It looks like some files were lost 
изображение

Comment (sungam3r) [reply src/cartservice/src/Program.cs:45]: Also I see this 
```
  <ItemGroup>
    <Protobuf Include="**\*.proto" GrpcServices="Both" />
  </ItemGroup>
```
in csproj but there is no any .proto file inside project file structure.

Comment (julianocosta89) [reply src/cartservice/src/Program.cs:45]: the `demo.proto` file is at the root of the project under `pb/`.
From the root folder you can run `make generate-protobuf` which is provided to generate protobuf files for all services.
This is documented here: https://opentelemetry.io/docs/demo/development/#generate-protobuf-files.

In `.NET` the following command is executed:
`cp -r "$base_dir"/pb/ ./src/protos/`

So you can basically copy the [`demo.proto` file](https://github.com/open-telemetry/opentelemetry-demo/tree/main/pb) to your project under `src/protos`.

Comment (sungam3r) [reply src/cartservice/src/Program.cs:45]: Thanks but it is still broken though.
изображение

`Startup` class was added in #23 and then (accidentally) removed in #489 . I do not see any ci workflow for running test project for .NET solution. Should I revert changes from #489 to keep `Startup.cs` file?

Comment (julianocosta89) [reply src/cartservice/src/Program.cs:45]: @cijothomas could you shed some light into this to @sungam3r ?

Comment (cijothomas) [reply src/cartservice/src/Program.cs:45]: Surprised that CI checks were not running the tests 👎 .
@sungam3r -To unblock you progress here, please comment out the tests temporarily, create an issue to fix them (and assign to me). I hope this is okay with the maintainers of the repo to unblock temporarily.

Comment (cartersocha) [reply src/cartservice/src/Program.cs:45]: Seems fine to me. Thanks for the help Cijo

Comment (sungam3r) [reply src/cartservice/src/Program.cs:53]: Done.

File: src/cartservice/src/Program.cs
Comment (julianocosta89) [inline src/cartservice/src/Program.cs:62]: Please remove the `ConsoleExporter`.

With this, the docker compose logs are bloated with cart-service entries:

```
cart-service             | GetCartAsync called with userId=
cart-service             | Activity.TraceId:            947e807dda97f7e45514843779a68bec
cart-service             | Activity.SpanId:             ddee9a80987ed0f0
cart-service             | Activity.TraceFlags:         Recorded
cart-service             | Activity.ParentSpanId:       455744d34f754ee7
cart-service             | Activity.ActivitySourceName: OpenTelemetry.Instrumentation.AspNetCore
cart-service             | Activity.DisplayName:        oteldemo.CartService/GetCart
cart-service             | Activity.Kind:               Server
cart-service             | Activity.StartTime:          2023-02-20T06:18:36.3621416Z
cart-service             | Activity.Duration:           00:00:00.0011445
cart-service             | Activity.Tags:
cart-service             |     net.host.name: cartservice
cart-service             |     net.host.port: 7070
cart-service             |     http.method: POST
cart-service             |     http.scheme: http
cart-service             |     http.target: /oteldemo.CartService/GetCart
cart-service             |     http.url: http://cartservice:7070/oteldemo.CartService/GetCart
cart-service             |     http.flavor: 2.0
cart-service             |     http.user_agent: grpc-node-js/1.6.7
cart-service             |     app.user.id:
cart-service             |     app.cart.items.count: 0
cart-service             |     http.status_code: 200
cart-service             |     rpc.system: grpc
cart-service             |     net.peer.ip: ::ffff:172.19.0.22
cart-service             |     net.peer.port: 49548
cart-service             |     rpc.service: oteldemo.CartService
cart-service             |     rpc.method: GetCart
cart-service             |     rpc.grpc.status_code: 0
cart-service             | Activity.Events:
cart-service             |     Fetch cart [02/20/2023 06:18:36 +00:00]
cart-service             | Resource associated with Activity:
cart-service             |     container.id: 81633081208a8d00bb50f85fa36ed32e43c459a8c10ca92abbf7532357c1c18d
cart-service             |     service.name: cartservice
cart-service             |     service.namespace: opentelemetry-demo
cart-service             |     telemetry.sdk.name: opentelemetry
cart-service             |     telemetry.sdk.language: dotnet
cart-service             |     telemetry.sdk.version: 1.4.0.788
```

File: src/cartservice/src/cartservice.csproj
Comment (julianocosta89) [inline src/cartservice/src/cartservice.csproj:10]: Also the `Console Exporter` dependency.

**Review by julianocosta89 — CHANGES_REQUESTED** [review]

File: src/cartservice/src/Program.cs
Comment (sungam3r) [reply src/cartservice/src/Program.cs]: done

**Review by julianocosta89 — APPROVED** [review]
LGTM

**Review by cijothomas — APPROVED** [review]

**Comment by cartersocha** [comment]
Please add a changelog entry @sungam3r and we're good to go!

**Comment by cijothomas** [comment]
> Please add a changelog entry @sungam3r and we're good to go!

Could we merge without changelog, and let changelog be sent separately? So as to unblock https://github.com/open-telemetry/opentelemetry.io/pull/2330

**Review by puckpuck — APPROVED** [review]
Tested running this branch and all LGTM!

**Pull request merged**
